src/routes/blog/post/implement-sign-in-with-github/+page.markdoc

```diff
@@ -0,0 +1,329 @@
+---
+layout: post
+title: How to implement Sign in with GitHub
+description: In this tutorial, you'll learn how to create a fully functional GitHub login system in your app.
+date: 2024-10-11
+cover: /images/blog/implement-sign-in-with-github/cover.png
+timeToRead: 10
+author: ebenezer-don
+category: auth
```

Comment:
category tutorial, auth doesn't exist in the list of categories (though it probably should)

Reply:
Can we add it or go with the usual "product"?

Reply:
Any reason you don't like tutorial? It fits either

Reply:
Oh, I thought it didn't exist. I'll change it

Reply:
@SnezhannaM Updated!